loader/app/loaders/loader_cclw_v2/load/main.py

```diff
@@ -62,6 +62,13 @@ async def load(ctx: Context, policies: PolicyLookup):
     logger.info(f"Done, imported {doc_count} docs from {len(policies.items())} actions")
 
 
+def warmup_local_caches():
```

Comment:
As some of the loader POSTs/uploads are now called from futures, best to warm up the cache outside of a possible stampeding herd.